File: game/src/spritesModule.cpp
#include "hModule/spritesModule.h"
#include "raylib.h"

/* Basic Structure of Class Variables for Characters or Objects:
  public:
        double posx;
        double posy;
        double widthSprite;
        double heightSprite;
        double widthCollider;
        double heightCollider;
        Texture2D image; 
        Rectangle collider;

    player::player(){
        posx = 0;
        posy = 0;
        widthSprite = 0;
        heightSprite = 0;
        widthCollider = 0;
        heightCollider = 0;
        image = LoadTexture("img/player.png");
        collider = Rectangle();
    } */

class player{
    public:
        double posx;
        double posy;
        double widthSprite;
        double heightSprite;
        double widthCollider;
        double heightCollider;
        Texture2D image; 
        Rectangle collider;

    player(){
        posx = 0;
        posy = 0;
        widthSprite = 0;
        heightSprite = 0;
        widthCollider = 0;
        heightCollider = 0;
        image = LoadTexture("img/player.png");
        collider = Rectangle();
    }
};

File: game/src/spritesModule.py

""" Basic Structure of Class Variables for Characters or Objects:
  self.posx =
        self.posy =
        self.widthSprite =
        self.heightSprite =
        self.widthCollider =
        self.heightCollider =  
        self.image = ray.load_texture("img/menú.png")
        self.collider = ray.Rectangle() """



import pyray as ray

class player():
    def __init__(self):
        self.posx = 0
        self.posy = 0
        self.widthSprite = 0
        self.heightSprite = 0
        self.widthCollider = 0
        self.heightCollider = 0
        self.image = ray.load_texture("img/menú.png")
        self.collider = ray.Rectangle()
        